### DISCUSSION TITLE
wget of release assets from a private repository no longer working

### DISCUSSION THREAD

**Discussion by jsbronder** [community/community#8383] [Other Feature Feedback, Questions, & Ideas] [answered]
Between 2am EST December 1st and 2am EST December 2nd, downloading assets from a private repository using wget stopped working.  When not setting `Accept: application/octet-stream`, I can still get the json object for the asset.

```
$ wget -O - \
    --header='authorization: token <filtered>' \
    https://api.github.com/repos/myorg/myrepo/releases/assets/1234

...

{
  "url": "https://api.github.com/repos/myorg/myrepo/releases/assets/1234",
  "id": 1234,
  ...
}
```
```
$ wget \
    --header='accept:application/octet-stream' \
    --header='authorization: token <filtered>' \
    https://api.github.com/repos/myorg/myrepo/releases/assets/1234

...

Resolving api.github.com (api.github.com)... 140.82.114.5
Connecting to api.github.com (api.github.com)|140.82.114.5|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://objects.githubusercontent.com/<filtered> [following]
...
Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.111.133, 185.199.109.133, 185.199.108.133, ...
Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.111.133|:443... connected.
HTTP request sent, awaiting response... 400 Bad Request
2021-12-03 11:20:46 ERROR 400: Bad Request.
```

**Comment by MylesBorins** [comment, marked as answer]
Is this still an issue?

**Reply by jsbronder** [reply]
No, it was resolved via a non-public support ticket: Ticket 1418421
<https://support.github.com/ticket/1418421>.  I'd assumed they'd update this
ticket.

> Dec 9, 2021, 8:31 AM UTC
>
> Hi <username>,
> 
> I heard back from the team and I understand this issue is now resolved.
> Could you please check again and let me know if you are still seeing the
> error?
> 
> Apologies for the inconvenience this may have caused you.
> 
> Regards,
> 
> Oluwaseun
> GitHub Support
> Dec 8, 2021, 11:18 AM UTC
> 
> Hi <username>,
> 
> Thanks for reaching out to GitHub Support!
> 
> I have reached out to the engineering team about this behaviour and will
> write back to you as soon as I have an update.
> 
> Regards,
> 
> Oluwaseun
> GitHub Support
> 
> *<username>* 
> 
> Dec 6, 2021, 5:17 PM UTC
> 
> Hi - we encountered the following bug in the release asset API (that was
> made last week) that has impacted our release build process:
> https://github.com/github/feedback/discussions/8383
> 
> I would like to determine if this is a bug or a permanent change in the
> assets API.
